docs/real-time-analytics/discounting/source.sql

```diff
@@ -0,0 +1,17 @@
+-- Example
+CREATE SOURCE CONNECTOR orders WITH (
+  'connector.class'          = 'PostgresSource',
+  'name'                     = 'recipe-postgres-orders',
+  'kafka.api.key'            = '<my-kafka-api-key>',
+  'kafka.api.secret'         = '<my-kafka-api-secret>',
+  'connection.host'          = '<my-database-endpoint>',
+  'connection.port'          = '5432',
+  'connection.user'          = 'postgres',
+  'connection.password'      = '<my-database-password>',
+  'db.name'                  = '<db-name>',
+  'table.whitelist'          = 'orders',
```

Comment:
@danicafine This may need to be updated?

Reply:
@danicafine IIUC `discount_codes` should be read from a table here as well?

Reply:
discount_codes doesn't have timestamps, wasn't sure how to resolve that. Two separate connectors or add timestamp to the table? Preference?

Reply:
1. IIUC `discount_codes` doesn't need timestamps, but it does still need to be read from a DB table somewhere
2. If you mean `orders` needs timestamps, you can refer to other recipes, e.g. https://github.com/confluentinc/ksqldb-recipes/blob/main/docs/cybersecurity/firewall-splunk/manual.sql, which use `UNIX_TIMESTAMP()`